omop_file_validator.py

```diff
@@ -359,21 +360,22 @@ def run_checks(file_path, f):
                                 submission_column, meta_column_type,
                                 submission_column_type, df)
                             if error_row_index:
-                                e = dict(message=MSG_INVALID_TYPE +
-                                         " line number " +
-                                         str(error_row_index + 1),
-                                         column_name=submission_column,
-                                         actual=df[submission_column]
-                                         [error_row_index],
-                                         expected=meta_column_type)
-                                result['errors'].append(e)
+                                # convert pd nan values to empty string
+                                df = df.replace(np.nan, '')
```

Comment:
You're right, I'm going to adjust this so it leaves it as `nan` objects!

Reply:
I tried to adjust and not change the DF at all, and the check would be:
`if not (df[submission_column][error_row_index] == pd.np.nan and not meta_column_required):`
However, it somehow doesn't catch the blank date and datetime in `device_exposure`. So I left in the `df.replace` I have it twice because I only want to change the df in those two checks so it doesn't affect any other part of the run outside of those `if`'s. Let me know if that makes sense?

Reply:
I see, that makes sense.